types/webpack/index.d.ts

```diff
@@ -9,12 +9,15 @@
 //                 Ahmed T. Ali <https://github.com/ahmed-taj>
 //                 Alan Agius <https://github.com/alan-agius4>
 //                 Spencer Elliott <https://github.com/elliottsj>
+//                 Jason Cheatham <https://github.com/jason0x43>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
Why is this necessary now to build using TS 2.2 instead of 2.7?

Reply:
I added a version line (there wasn't one to begin with) because the default version used in CI is 2.1, which doesn't understand `object` (which I was using in the self-tests). However, specifying 2.2 here breaks other packages that depend on webpack, like case-sensitive-paths-webpack-plugin, because 2.2 is more recent than the default 2.1. Using anything higher than 2.2 fails local linting. (It may still pass CI, but then what's the point of the local linter?)

My latest version just removes the version line again and uses the specific `RawSourceMap` type in the tests rather than generic `object`.